src/main.c

```diff
@@ -116,6 +165,7 @@ int main(void)
 		int right = timer_get_counter(TIM4);
 		for (int i = 0; i < 800000; i++)
 			__asm__("nop");
+		printf("Left: %d, Right: %d\n", left, right);
```

Comment:
I think that modifying standard printf function to send data by USART could be confusing. We could create a new function with a different name (printf_usart?) that includes printf and this USART feature.

Reply:
Created issue https://github.com/Theseus/theseus/issues/28.